### DISCUSSION TITLE
The "Add IP Manually" option

### DISCUSSION THREAD

**Discussion by ITZTHATPLAYER** [LBPUnion/ProjectLighthouse#522] [Ideas]
I say you should add this feature because there is probably many people having problems to reach to the servers, I even have this problem too on my other ps3 and on rpcs3

an example is that someone is trying to use a ps3, or use a pc on the rpcs3 emulator to play Little Big Planet 1 - 2 - 3 online, some people cannot play online because maybe they are getting an issue whereas they cannot see their IP on the "authentications" tab, this could probably save you guys a lot of time off of helping others for their problems, how this is supposed to work is that if the player is not seeing their IP being shown as they have patched their games properly without any problem, then they could just enter their IP in a textbox where it can authenticate that IP without it automatically popping up as it normally does

as you are looking at this suggestion, many people are waiting or even giving up on trying to get back on lbp online because many cant be asked to even fix the problem because their IP is not showing, but with My Suggestion "Add IP manually" would probably fix other people's frustration over the IP not getting sent to the authentications tab.

This IP problem is a problem which happens both on all of the LBP games, specific ps3(s) in your house connected to the internet, pc emulation (rpcs3)

**Comment by m88youngling** [comment]
I appreciate your suggestion, but I'm not sure this is really necessary. I'm guessing you're not in the beta testing program, but this is the general process we recommend to players via our currently private tutorials (they will be publicized once we hit public beta):

1. You patch your game to the URL of the server you want to connect to in your EBOOT. 

2. You create and sign into an account on the associated website of that server. 

3. Attempt to connect to the server, you get an error code 403. If you patched correctly it sends an authentication request to the server. 

4. Check the authentication tab. If you see an authentication request from your console's IP address, congrats! You did it.

If you don't see an authentication request when you believe you should, there are two possibilities:

1. You didn't patch correctly, check the URL that you used. Some users have reported accidentally using a period at the end of the URL. 

2. There may be an issue with your account. Check to make sure that your PSN or RPCN name matches the website account name EXACTLY. 

Based on this troubleshooting, the current system is sufficient. The problem can be found and corrected fairly easily

**Reply by ITZTHATPLAYER** [reply]
or it could be your dns if you jailbroken your ps3 to cfw in 2022, which I fixed but its just rpcs3 not working for me not sending me the IP to authenticate

**Reply by m88youngling** [reply]
Are you trying to connect to your own private server or are you attempting to connect to Beacon? We have a lot of beta testers who run the latest version of Evilnat CFW and none of them have had an issue related to DNS settings.

**Reply by ITZTHATPLAYER** [reply]
ive already connected on cfw evilnat and I had to copy something for the dns because I was following what was in the video, then I followed the video's instructions furthermore
until I was done, I forgot to revert my dns back to stock

**Reply by ITZTHATPLAYER** [reply]
so I reverted it and it works but its just rpcs3

**Reply by m88youngling** [reply]
We don't have any official videos about this, so be careful what you look at. There will be more formal tutorials once we hit public beta. None of us on our team have ever given any advice about DNS settings as far as I am aware. As long as your DNS is set so that it can communicate with an external server over the Internet, then it should work. Typing your IP manually really wouldn't help you fix this since it would require fixing the issue with your DNS settings first I think.

Back to the original topic of your suggestion, I actually think there is some risk for security vulnerabilities with punching in an IP manually. It's more secure if the server has to get your IP automatically to prevent abuse.

**Reply by ITZTHATPLAYER** [reply]
I can send the vid rn

**Reply by ITZTHATPLAYER** [reply]
https://www.youtube.com/watch?v=P3yWPPqGvx8

**Comment by ITZTHATPLAYER** [comment]
also I set my dns back to auto so its working
its just rpcs3 not working

**Reply by m88youngling** [reply]
Since this has shifted into a troubleshooting discussion I am gonna go ahead and lock this, if you need any more help please refer to the following, 

1. We provide support for our Project Lighthouse beta testing team during private beta. You can enlist in the private beta test waitlist by joining our Discord server. You'll need the citizen role from the #welcome channel and the USCEnlist role from the #settings channel. 

2. You could also wait for publicly available tutorials we plan on publishing in the near future.
